content/en/security/application_security/enabling/compatibility/nodejs.md

```diff
@@ -6,46 +6,163 @@ type: multi-code-lang
 code_lang_weight: 50
 ---
 
+## ASM capabilities 
+
+The following ASM capabilities are supported in the Node.js library, for the specified tracer version:
+
+| ASM capability                   | Minimum NodeJS tracer version |
+| -------------------------------- | ----------------------------|
+| Threat Detection  | 3.13.1|
```

Comment:
not sure what that version is supposed to represent

Reply:
this is the minimum node tracer version where all threat detection rules are supported. Do you know if the version is earlier than 3.13.1?